scripts/validationAndPreparation.sh

```diff
@@ -1,45 +1,62 @@
+#!/bin/bash
+alias connect_server='sftp -i /root/.ssh/id_rsa_worker validationworker@trusted-setup.staging.gnosisdev.com'
 
-FILES=`lftp -e "set ssl:key-file ~/.ssh/id_rsa; cls; bye" sftp://josojo:@trusted-setup.staging.gnosisdev.com`
-if [[ -z "${NEWESTDATE}" ]]; then
-  NEWESTDATE=0
+if [[ -z "${DATE_OF_NEWEST_CONTRIBUTION}" ]]; then
```

Comment:
I don't wanna over write this variable in case we want to start with a DATE_OF_NEWEst_CONTRITBION from the last instance. This helps so that new instances do not have to re-evaluate all the contributions

Reply:
My argument is that these variables will always be set, so there is never a need to write them inside the script. For the very first run we can just write the initial values into the project_env file.